workflows-scripts/List-App/list_app.jmx

```diff
@@ -0,0 +1,622 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.0 r1840935">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="master-script" enabled="true">
```

Comment:
could you please rename master-script test name to avoid confusion in future test runnings?

Reply:
Changed